src/theme/typography.js

```diff
@@ -0,0 +1,107 @@
+// override mui default
+// https://material-ui.com/customization/default-theme/
+
+const typography = {
+  htmlFontSize: 16,
+  fontFamily: "'Inter', 'Arial', sans-serif",
+  fontSize: 16,
+  fontWeightLight: 300,
+  fontWeightRegular: 400,
+  fontWeightMedium: 500,
+  fontWeightBold: 600,
+  h1: {
+    fontFamily: "'Inter', 'Arial', sans-serif",
+    fontWeight: 600,
+    fontSize: '1.75rem', // 28px
+    lineHeight: 1.5,
```

Comment:
Same with `lineHeight`.

```
const lineHeight = 1.5;
```

Reply:
I've left it for now as it's not uncommon to have different line heights for each h1 - h6